### ISSUE TITLE
Give feedback on improvements or changes necessary for AmiGO use case of pathway widget

### ISSUE THREAD

**Issue by kltm** [geneontology/amigo#679]
As we are adding a new use case for the pathway widget (AmiGO), there may be some changes or adjustments that are necessary. Those can be collected and discussed here.

Tagging @vanaukenk and @ukemi

**Comment by ukemi** [comment]
I've made a few straw-man mock ups for the way we should display metadata with the models.
https://docs.google.com/presentation/d/1mPBRXrvZnMOFgmXMop_SQ3lZNnp97dUQ2XMao1f1VTo/edit#slide=id.p

**Comment by pkalita-lbl** [comment]
Yep, reviewed and left comments on the slides. Adjusting the contents of the Model Information section is easy-peasy. I can get to that next week.

**Comment by ukemi** [comment]
Would like to have @vanaukenk have a look as well.

**Comment by vanaukenk** [comment]
Thanks @pkalita-lbl 

@ukemi and I just looked over the slides together and I made a few additional comments, mainly about:

1) Comments - we're not sure these are always intended to be public, so perhaps we could use the wasDerivedFrom tag to at least capture when a model was sourced/copied from another model without having to display all the comments.

2) The gene link outs could stay internally to AmiGO, but we should talk with the GO PIs about possibly linking out to Alliance pages, whenever possible.  @kltm may also be able to comment more here, as linking out to the Alliance has come up in other contexts.

**Comment by kltm** [comment]
@vanaukenk Just commenting on number "2". It is not possible to blanket link to the Alliance until they fix routing, and routing should be taken care of by AmiGO's overall linking strategy (i.e. whatever derivation of db-xrefs.yaml is available in you're particular spot)--we wouldn't want inconsistent linking. I think it's fair at this point to link internally for GPs in AmiGO or to directly link out to whatever the external link is given (the MODs for MODs). That said, I think it's important at this point to be able to "escape" the model landing pages in some way to get into the rest of AmiGO and browsing; that could be through GP links, term links, or some other mechanism. (This will improve in the next iteration as we add in things like search an closures, but that's for the next round.)

**Comment by pkalita-lbl** [comment]
`amigo-staging` has been updated with the suggested changes to the Model Information section. For example: https://amigo-staging.geneontology.io/amigo/model/62f58d8800000934

Nothing has changed with respect to the GO-CAM viewer widget itself. I did make some issues in this project: https://github.com/orgs/geneontology/projects/140/views/1. A few of those would allow AmiGO to use future versions of the widget, as opposed to the slightly older version it needs to use for now.
